src/sorting.jl

```diff
@@ -390,5 +390,5 @@ function quicksort!(c :: CuArray{T}) where T
     end
 
     b_sums = CuArray{Int32}(undef, ceil_div(N, 32))
-    CUDA.@sync(@cuda config=get_config  qsort_kernel(c, Int32(0), N, b_sums, Val(true), Int8(1)))
+    CUDA.@sync(@cuda threads=64 shmem=(sizeof(T) + 4)*64  qsort_kernel(c, Int32(0), N, b_sums, Val(true), Int8(1)))
```

Comment:
See here for an example on using the occupancy API with a shmem callback: https://github.com/JuliaGPU/CUDA.jl/blob/bc0184eaba90c303e73c44a1768b6b7cb36ed669/src/mapreduce.jl#L196-L197